test/compress/export.js

```diff
@@ -154,3 +154,46 @@ issue_2134_2: {
         Foo.prototype = {};
     }
 }
+
+redirection: {
+    mangle = {
+        toplevel: true,
+    }
+    input: {
+        let foo = 1;
+        export { foo as var };
+        export { foo as var } from "module.js";
+    }
+    expect: {
+        let o = 1;
+        export { o as var };
+        export { foo as var } from "module.js";
+    }
+}
+
+keyword_invalid: {
+    input: {
+        export { default };
+        export { default as Alias };
+        export { default as default };
+    }
+    expect: {
+        export { default };
+        export { default as Alias };
+        export { default as default };
+    }
+}
+
+keyword_valid: {
+    input: {
+        export { default } from "module.js";
+        export { default as Alias } from "module.js";
+        // duplicated export of the "default" alias
```

Comment:
@kzc https://github.com/alexlamsl/UglifyJS2/commit/a1947f143bf33b5cb29ad42f76064d5e3e3f22a6#commitcomment-22708122 is done here.